packages/plugin-mentions/web/src/tiptap/suggestion.ts

```diff
@@ -70,6 +82,11 @@ export default (
             return true;
           }
 
+          if (props.event.key === 'Enter') {
+            const { items, query } = component.props;
+            shouldEndMentioningProcess(query, items) && this.onExit();
+          }
```

Comment:
```suggestion
          if (shouldEndMentioningProcess(query, items)) {
              this.onExit();
          }
```

check & end mention process on every char typed and not only on Enter.
This need to be the first condition because I think arrow-up\arrow-down will also not work